packages/@divvi/mobile/src/onboarding/welcome/Welcome.tsx

```diff
@@ -95,10 +94,7 @@ export default function Welcome() {
   const assetsConfig = getAppConfig().themes?.default?.assets
 
   const Logo = assetsConfig?.welcomeLogo ?? WelcomeLogo
```

Comment:
Looks like this can be removed now and use `WelcomeLogo` directly, right?

Reply:
oh yeah!